core/federated/RTI/rti.Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG BASEIMAGE=alpine:latest
 FROM ${BASEIMAGE} AS builder
 COPY . /lingua-franca
 WORKDIR /lingua-franca/core/federated/RTI
-RUN set -ex && apk add --no-cache gcc musl-dev cmake make && \
+RUN set -ex && apk add --no-cache gcc musl-dev cmake make git && \
```

Comment:
We need git because the build process uses it to get the commit ID of the build which is printed when -v/--version is passed